postings/urls.py

```diff
@@ -15,4 +13,6 @@
     ])),
     path('/datail/<int:posting_id>', PostView.as_view()),
     path('/like', LikeView.as_view()),
-]
+    path('/<int:posting_id>/comment', CommentView.as_view()),
+    path('/<int:comment_id>/delete', CommentView.as_view()),
```

Comment:
현재 엔드포인트 주소가 `/postings/1/delete`와 같이 되어 있습니다. 이와 같은 경우는 comment를 삭제하는 것이 아니라 1번 posting을 삭제하는 것으로 보여집니다. 아래와 같이 수정해주세요!
```suggestion
    path('/comment/<int:comment_id>', CommentView.as_view()),
```